### PR TITLE
Internal: Aligned release tools to the latest CKEditor 5 dev.

### PR DESCRIPTION
**Pull request by martnpaneq**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Aligned release tools to the latest CKEditor 5 dev. Closes ckeditor/ckeditor5-react#378.

---

### Additional information

*For example – encountered issues, assumptions you had to make, other affected tickets, etc.*

### CODE REVIEW

File: scripts/preparepackages.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+		title: 'Cleaning-up.',
+		task: () => {
+			return releaseTools.cleanUpPackages( {
+				packagesDirectory: 'release',
+				packageJsonFieldsToRemove: [
+					// TODO: Preserve `scripts.postinstall`.
+					// See: https://github.com/ckeditor/ckeditor5/issues/14318.
+					'devDependencies'
+				]
+			} );
```
Comment (psmyrek) [inline scripts/preparepackages.js:69]: Do we need the `postinstall` script in a package published on npm?

File: package.json [hunk 2]
```diff
@@ -114,7 +117,10 @@
   "homepage": "https://github.com/ckeditor/ckeditor5-react",
   "files": [
     "dist",
-    "scripts/postinstall.js"
+    "scripts/postinstall.js",
+    "README.md",
+    "CHANGELOG.md",
```
Comment (psmyrek) [inline package.json:122]: Previously, we did not publish the `CHANGELOG.md` - see https://www.npmjs.com/package/@ckeditor/ckeditor5-react?activeTab=code. Is it needed on npm?

**Review by psmyrek — COMMENTED** [review]
Changes work fine 🎉

Maybe we could remove the `dist` directory from the GitHub and add it to `.gitignore`?
Also, there are some remarks to `dist` directory in the `README.md` that probably should be updated.

**Review by psmyrek — APPROVED** [review]
LGTM 🎉

**Review by pomek — APPROVED** [review]
LGTM.

**Pull request merged**